pkg/kn/flags/podspec.go

```diff
@@ -163,3 +170,165 @@ func (p *PodSpecFlags) AddFlags(flagset *pflag.FlagSet) []string {
 	flagNames = append(flagNames, "user")
 	return flagNames
 }
+
+func (p *PodSpecFlags) ResolvePodSpec(podSpec *corev1.PodSpec, cmd *cobra.Command) error {
```

Comment:
Thank you for the suggestion.
Here, I guess I have to use `cmd`, because there are some statement in this function to print to `cmd` output, for example:
```
line 237: fmt.Fprintf(cmd.OutOrStdout(), "\nWARNING: flags --limits-cpu / --limits-memory are deprecated and going to be removed in future release, please use --limit instead.\n\n")
```

Reply:
We can check for such deprecated flags in `pkg/kn/commands/service/configuration_edit_flags.go` before invoking `ResolvePodSpec`.

Reply:
Thank you. I've updated based on your comments.